ZCRMiOS/Classes/ZCRMModule.swift

```diff
@@ -1019,31 +1019,24 @@ public class ZCRMModule : ZCRMEntity
         } )
     }
     
-    public func deleteTag( tag : ZCRMTag, completion : @escaping ( APIResponse?, Error? ) -> () )
+    public func addTags( recordIds : [Int64], tags : [ZCRMTag], completion : @escaping( [ZCRMRecord]?, BulkAPIResponse?, Error? ) -> () )
```

Comment:
Why is addTags returning [ZCRMRecord] ? shouldn't it be [ZCRMTag] ?

Reply:
Discussed